internal/dwarf/frame/table.go

```diff
@@ -9,6 +9,9 @@ import (
 	"github.com/parca-dev/parca-agent/internal/dwarf/util"
 )
 
+// TODO(javierhonduco): Add source
+const RBPRegister = 6
```

Comment:
TODO: Remove.

Reply:
Oops, did not `git add` that file... Will remove it later, sorry about this!